src/Microsoft.Diagnostics.Monitoring.WebApi/Metrics/JsonCounterLogger.cs

```diff
@@ -35,34 +36,93 @@ public JsonCounterLogger(Stream stream, ILogger logger)
 
         protected override async Task SerializeAsync(ICounterPayload counter)
         {
-            await _stream.WriteAsync(JsonSequenceRecordSeparator);
+            if (counter is ErrorPayload errorPayload)
+            {
+                Logger.LogWarning(errorPayload.ErrorMessage);
```

Comment:
This needs to be moved to LoggingExtensions

Reply:
Done - the error messages themselves come with their own explanatory text, so I omitted saying something like, "The instrument produced an error" before it because it was redundant. I can change this if desired.